scripts/metrics.py

```diff
@@ -136,6 +139,11 @@
         # legacy check
         if ('emb' in adata_int.uns) and (adata_int.uns['emb']):
             adata_int.obsm["X_emb"] = adata_int.obsm["X_pca"].copy()
+        # fix for unintegrated metrics
+        if "X_emb" not in adata_int.obsm:
```

Comment:
why is this necessary? Can't you just encode unintegrated not as an embedding methods, but as `type=full`?